game-app/game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -395,15 +395,14 @@ private static String newLabelText(final DownloadFileDescription map) {
                         .append(")"));
       }
     } else {
+      sb.append(doubleSpace).append(" (");
       try {
-        sb.append(doubleSpace)
-            .append(" (")
-            .append(newSizeLabel(Files.size(map.getInstallLocation().get())))
-            .append(")");
+        sb.append(newSizeLabel(Files.size(map.getInstallLocation().get())));
       } catch (final IOException e) {
         log.warn("Failed to read file size", e);
```

Comment:
I was more referring for there to be an update to the logger mesage. The user will be presented with a pretty vague error message here, if downloading multiple maps then it won't be clear what happened or what they should do.

Reply:
Ah my bad. I don't think this will happy very often. I didn't check but unless you change file permission while TripleA is running I don't think we will ever hit this condition. But maybe the best decision is to decrease the logger level so it's only logged but without message to the user?

Reply:
It's tempting to ignore outright, hard to judge best course of action. I think when in doubt slap some additional information in and move on.